contracts/src/v0.8/ccip/pools/GHO/UpgradeableTokenPool.sol

```diff
@@ -250,7 +257,8 @@ abstract contract UpgradeableTokenPool is IPool, OwnerIsCreator, IERC165 {
   /// is a permissioned onRamp for the given chain on the Router.
   modifier onlyOnRamp(uint64 remoteChainSelector) {
     if (!isSupportedChain(remoteChainSelector)) revert ChainNotAllowed(remoteChainSelector);
-    if (!(msg.sender == s_router.getOnRamp(remoteChainSelector))) revert CallerIsNotARampOnRouter(msg.sender);
+    if (!(msg.sender == getProxyPool() || msg.sender == s_router.getOnRamp(remoteChainSelector)))
+      revert CallerIsNotARampOnRouter(msg.sender);
```

Comment:
wrap with `{}` if cannot fit in a single line

Reply:
[1de6efb](https://github.com/aave/ccip/pull/18/commits/1de6efbaef65b2f35d2cf222f122d48a7bc0d9d6)